test-engineering/load-tests/load_test_setup.sh

```diff
@@ -36,7 +35,7 @@ echo -e "==================== Set SCOPE "
 export SCOPE=$SCOPE
 
 echo -e "==================== Refresh Kubeconfig at path ~/.kube/config "
-$GCLOUD container clusters get-credentials $CLUSTER --zone $ZONE --project $GOOGLE_CLOUD_PROJECT
+$GCLOUD container clusters get-credentials $CLUSTER --region $REGION --project $GOOGLE_CLOUD_PROJECT
```

Comment:
Uh, I had to make this change to make the load tests work for me personally, since I was using a different region. However, none of the changes to the load test files should be in this PR. Oops.

Reply:
Happens to the best of us!